tests/contrib/httpx/test_httpx.py

```diff
@@ -89,6 +89,7 @@ async def test_get_200(snapshot_context):
             assert resp.status_code == 200
 
 
+@flaky(until=1706677200, reason="flaky errors with no error logs, need to investigate")
```

Comment:
I set the epoch timestamp to this value because I saw it was a common one, but looks like it's all the way out in Jan 2025. Couldn't find docs on what the expectations are for setting these timestamps, so if anyone could share (if we have them) that'd be great.

Reply:
I'll write up some documentation about how to use the flaky decorator.